content/docs/manifest.json

```diff
@@ -344,6 +344,10 @@
             {
               "title": "trust-manager",
               "path": "/docs/projects/trust-manager.md"
+            },
+            {
+              "title": "trust-manager API Docs",
+              "path": "/docs/projects/trust-manager-api-ref.md"
```

Comment:
note: I couldn't decide between /projects and /reference

Reply:
we might need a API ref page for approval-policy too

Reply:
I would like to have it all in a separate subsection (eg /docs/projects/reference or /docs/reference/projects)

Reply:
I suppose it depends whether we are trying to make all these projects appear as an integrated suite of software or whether we'd rather keep each project more self container.
I think for now, I'd prefer it in `/docs/projects/trust-manager/api-reference.md`. My 2c

Reply:
>  I think for now, I'd prefer it in /docs/projects/trust-manager/api-reference.md. My 2c

Totally reasonable, yeah. I'll go for making trust-manager its own section, then, as in:

```text
Projects
-> approver-policy
-> ...
-> trust-manager
---> Introduction
---> API Reference Docs
```

(I think it's a valuable long-term goal for trust-manager to be able to run entirely independently of cert-manager, to the point where it could live in its own independent docs site like `https://trust-manager.cert-manager.io`. I think a lot of the other subprojects won't reasonably exist independently of cert-manager, but trust-manager is useful even if you never install cert-manager)